src/main/java/soonwee/duke/Storage.java

```diff
@@ -57,6 +78,11 @@ public void readFile() {
         }
     }
 
+    /**
+     * Process data in file and add them into memory task list.
+     *
+     * @param data input data taken from command.
+     */
```

Comment:
Maybe you can consider to change Process into Processes. Just one tiny suggestion.